docs/csharp/language-reference/keywords/default-values-table.md

```diff
@@ -46,10 +34,36 @@ Remember that using uninitialized variables in C# is not allowed.
 |[ulong](ulong.md)|0|
 |[ushort](ushort.md)|0|
 
+## Remarks
+
+You cannot use uninitialized variables in C#. You can initialize a variable with the default value of its type. You also can use the default value of a type to specify the default value of a method's [optional argument](../../programming-guide/classes-and-structs/named-and-optional-arguments.md#optional-arguments).
+
+Use [default value expression](../../programming-guide/statements-expressions-operators/default-value-expressions.md) to produce the default value of a type as the following example shows:
```

Comment:
two nits that I missed in my earlier review:
- Use --> Use the
- comma before "as"